corehq/apps/users/migrations/0059_copy_transient_invitation_supply_point.py

```diff
@@ -1,14 +1,73 @@
 # Generated by Django 3.2.25 on 2024-04-01 16:59
+import sys
+import traceback
 
-from django.core.management import call_command
+from django.core.management import call_command, get_commands
 from django.db import migrations
 
+from corehq.apps.domain_migration_flags.api import get_migration_complete, ALL_DOMAINS
 from corehq.util.django_migrations import skip_on_fresh_install
+from corehq.apps.users.management.commands.copy_invitation_supply_point import MIGRATION_SLUG
+
+COUNT_ITEMS_TO_BE_MIGRATED = "SELECT COUNT(*) FROM users_invitation WHERE users_invitation.location_id is null AND users_invitation.supply_point IS NOT NULL"
+GIT_COMMIT_WITH_MANAGEMENT_COMMAND = "c3f318f328372e904f4b1db42177309211ceb332"
+AUTO_MIGRATE_ITEMS_LIMIT = 200000
+AUTO_MIGRATE_COMMAND_NAME = "copy_invitation_supply_point"
+AUTO_MIGRATE_FAILED_MESSAGE = """
+This migration cannot be performed automatically and must instead be run manually
+before this environment can be upgraded to the latest version of CommCare HQ.
+Instructions for running the migration can be found at this link:
+
+TO BE UPDATED
```

Comment:
can you update this please? Thanks!

Reply:
[b822aaf](https://github.com/dimagi/commcare-hq/pull/34525/commits/b822aafdd3b2f5997e8d4671cfcc2e9cab5f88e5)